Also needs to check if _MSC_VER is defined at all.

--- es-core/src/Util.h	2016-03-13 20:31:00.000000000 +0100
+++ es-core/src/Util.h	2016-09-15 10:33:18.964427356 +0200
@@ -15,7 +15,7 @@
 Eigen::Vector3f roundVector(const Eigen::Vector3f& vec);
 Eigen::Vector2f roundVector(const Eigen::Vector2f& vec);
 
-#if _MSC_VER < 1800
+#if defined(_MSC_VER) && (_MSC_VER < 1800)
 float round(float num);
 #endif /* _MSC_VER */
 
--- es-core/src/Util.cpp	2016-03-13 20:31:00.000000000 +0100
+++ es-core/src/Util.cpp	2016-09-15 10:36:10.972624004 +0200
@@ -32,14 +32,14 @@
 }
 
 
-#if _MSC_VER < 1800
+#if defined(_MSC_VER) && (_MSC_VER < 1800)
 float round(float num)
 {
 	return (float)((int)(num + 0.5f));
 }
 #endif
 
-#if _MSC_VER >= 1700
+#if defined(_MSC_VER) && (_MSC_VER >= 1700)
 FILE iob[] = {*stdin, *stdout, *stderr };
 FILE * __iob_func(void)
 {
